integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/mailto/transport/receiver/test/MailToTransportFolderTestCase.java

```diff
@@ -36,17 +39,20 @@
 
 public class MailToTransportFolderTestCase extends ESBIntegrationTest {
     private String emailSubject;
-    private static LogViewerClient logViewerClient;
+    private static CarbonLogReader carbonLogReader;
     private static GreenMailClient greenMailClient;
     private static GreenMailUser greenMailUser;
 
     @BeforeClass(alwaysRun = true)
     public void initialize() throws Exception {
         super.init();
-        loadESBConfigurationFromClasspath(
-                File.separator + "artifacts" + File.separator + "ESB" + File.separator + "mailTransport"
-                        + File.separator + "mailTransportReceiver" + File.separator + "mail_transport_folder.xml");
-        logViewerClient = new LogViewerClient(contextUrls.getBackEndUrl(), getSessionCookie());
+        OMElement mailToProxyOMElement = AXIOMUtil.stringToOM(FileUtils.readFileToString(new File(
+                getESBResourceLocation() + File.separator + "mailTransport" + File.separator +
+                        "mailTransportReceiver" + File.separator + "mail_transport_folder.xml")));
```

Comment:
`+` should precede in the new line according to the followed code style. Maybe the IDE config is not correct.

Reply:
Done in #575